stomp/adapter/ws.py

```diff
@@ -212,7 +220,8 @@ def receive(self):
         :rtype: bytes
         """
         try:
-            return self.socket.recv().encode()
+            opcode, data = self.socket.recv_data()
```

Comment:
doesn't this imply its assuming its binary mode now?

Reply:
Looking at what the `WebSocket.recv` method does, if the opcode is text, then it returns a string, and if the opcode is binary, then it returns bytes:

https://github.com/websocket-client/websocket-client/blob/master/websocket/_core.py#L353

So the current behaviour when we receive a text message is to:

* call `decode` to turn the bytes into a utf-8 string (in `WebSocket.recv`)
* encode the string as bytes (in `WSTransport.receive`)

Whereas, if we call `recv_data` instead, we don't need to encode/decode at all.

Reply:
got it. thanks for validating that.